addons/product/models/product_attribute_value.py

```diff
@@ -43,6 +43,9 @@ def _get_default_color(self):
             " to display the color if the attribute type is 'Color'.")
     display_type = fields.Selection(related='attribute_id.display_type')
     color = fields.Integer(string="Color Index", default=_get_default_color)
+    image_1920 = fields.Image(
+        string="Image",
+        help="If an image is set, the color will not be used on eCommerce")
```

Comment:
Discussed with PO, better to explicitly mention what the image will be used for
```suggestion
        help="You can upload an image that will be used as the color of the attribute value.")
```